web/packages/teleport/src/services/history/history.ts

```diff
@@ -53,24 +53,39 @@ const history = {
     window.location.reload();
   },
 
-  goToLogin(rememberLocation = false) {
-    let url = cfg.routes.login;
+  goToLogin({ rememberLocation = false, withAccessChangedMessage = false }) {
+    const params: string[] = [];
+
+    // withAccessChangedMessage determines whether the login page the user is redirected to should include a notice that
+    // they were logged out due to their roles having changed.
+    if (withAccessChangedMessage) {
+      params.push('access_changed');
```

Comment:
should `access_changed` be an enum? or a const? since we use it to look for it in `hasAccessChangedParam`